applications/ray/kuberay-tpu-webhook/main.go

```diff
@@ -86,7 +86,7 @@ func getNumTPUHostsFromTopology(topology string, acceleratorType string) (int32,
 	for i := 0; i < len(topologyVals); i++ {
 		dim, err := strconv.Atoi(topologyVals[i])
 		if err != nil {
-			klog.Errorf("Invalid topology: %s", err)
+			klog.Errorf("RayCluster: %s | Invalid topology: %s", clusterName, err)
```

Comment:
You probably want to log the namespace too `<namespace>/<name>`

Reply:
Done.